docs/getting-started/usage.md

```diff
@@ -106,14 +106,39 @@ The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/cert
 ### Run Edge
 
 #### Deploy the Edge node
-We have provided a sample node.json to add a node in kubernetes. Please make sure edge-node is added in kubernetes. Run below steps to add edge-node.
+KubeEdge supports auto registration of edge-nodes but we have provided a sample node.json to manually add a node in kubernetes if required. Run below steps to add edge-node.
+
++ Copy the `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` file and change `metadata.name` to the name of the edge node
+    
+    ```shell
+        mkdir ~/cmd/yaml
+        cp $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json ~/cmd/yaml
+    ```
+    
++ Make sure role is set to edge for the node. For this a key of the form `"node-role.kubernetes.io/edge"` must be present in `labels` tag of `metadata`.
++ Please ensure to add the label `node-role.kubernetes.io/edge` to the `build/node.json` file.
+
```

Comment:
Why support manual registration since it supports automatic registration?

Reply:
We can still allow manual addition of nodes for the administrators to additionally configure nodes by adding labels and other fields. By default only role label and name label is added. But if someone wants to add nodes with common labels so that they can be grouped together, it can be useful.